From 3bd0ac580250d3640a1a7955f01c7f8eb9f543a4 Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Thu, 1 Nov 2018 16:47:59 +0800
Subject: [PATCH 644/767] ptp: add PTP_1588_CLOCK_QORIQ for QorIQ platform

PTP_1588_CLOCK_QORIQ is to enable ptp_qoriq driver for
DPAA/eTSEC 1588 timer.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/ptp/Kconfig | 17 +----------------
 1 file changed, 1 insertion(+), 16 deletions(-)

diff --git a/drivers/ptp/Kconfig b/drivers/ptp/Kconfig
index b2d84775d2ae..0faa6d8c468b 100644
--- a/drivers/ptp/Kconfig
+++ b/drivers/ptp/Kconfig
@@ -43,7 +43,7 @@ config PTP_1588_CLOCK_DTE
 
 config PTP_1588_CLOCK_QORIQ
 	tristate "Freescale QorIQ 1588 timer as PTP clock"
-	depends on GIANFAR
+	depends on GIANFAR || FSL_DPAA_ETH || FSL_SDK_DPAA_ETH
 	depends on PTP_1588_CLOCK
 	default y
 	help
@@ -55,21 +55,6 @@ config PTP_1588_CLOCK_QORIQ
 	  To compile this driver as a module, choose M here: the module
 	  will be called ptp_qoriq.
 
-config PTP_1588_CLOCK_DPAA
-	tristate "Freescale DPAA as PTP clock"
-	depends on FSL_SDK_DPAA_ETH
-	select PTP_1588_CLOCK
-        select FSL_DPAA_TS
-        default n
-	help
-	  This driver adds support for using the DPAA 1588 timer module
-	  as a PTP clock. This clock is only useful if your PTP programs are
-	  getting hardware time stamps on the PTP Ethernet packets
-	  using the SO_TIMESTAMPING API.
-
-	  To compile this driver as a module, choose M here: the module
-	  will be called dpaa_ptp.
-
 config PTP_1588_CLOCK_DPAA2
 	tristate "Freescale DPAA2 as PTP clock"
 	depends on FSL_DPAA2_ETH
-- 
2.17.0

